modules/creating-custom-links.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * web-console/customizing-the-web-console.adoc
+
+[id="creating-custom-links_{context}"]
+= Creating custom links in the web console
+
+.Procedure
+
+. From *Administration* -> *Custom Resource Definitions*, click on
+*ConsoleLink*.
+
+. Click *YAML* and edit the file:
++
+----
```

Comment:
Sorry, just getting to this to review.  Just wanted to note that the `code` here is `golang`, which is from `openshift/api` to describe the API objects that are created.  However, I think you may want to include `yaml`.

The `yaml` that a customer might add for creating a link would look more like:

```yaml
apiVersion: console.openshift.io/v1
kind: ConsoleLink
metadata:
  name: example
spec:
  href: 'https://www.example.com'
  location: HelpMenu
  text: Link 1
```
The `golang` API docs (https://github.com/openshift/api/tree/master/console/v1) are really helpful for knowing what options are available if you are interested in reading the comments between the lines.  For example:

```golang
// location determines which location in the console the link will be appended to.
Location ConsoleLinkLocation `json:"location"`
```
Which just lets you know you can use `location`, with a value like `HelpMenu`.